java/AvroOneTopicManySubjects/producer-app/pom.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.amazonaws</groupId>
+    <artifactId>producer-app</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <buildDirectory>${project.basedir}/target</buildDirectory>
+        <jar.finalName>${project.name}</jar.finalName>
+        <target.java.version>11</target.java.version>
+        <maven.compiler.source>${target.java.version}</maven.compiler.source>
+        <main.class>com.amazonaws.services.msf.Sensors</main.class>
+
+        <avro.version>1.10.2</avro.version>
+        <confluent.version>7.9.0</confluent.version>
+        <kafka.version>3.9.0</kafka.version>
+        <slf4j.version>1.7.32</slf4j.version>
+        <log4j.version>2.17.2</log4j.version>
+        <junit.version>5.9.1</junit.version>
+        <commons.version>1.9.0</commons.version>
+    </properties>
+
+    <repositories>
```

Comment:
This is not required
`io.confluent:kafka-avro-serializer` are in [Maven central](https://mvnrepository.com/artifact/io.confluent/kafka-avro-serializer)